proto/confidentialtransfers/query.proto

```diff
@@ -3,26 +3,38 @@ package seiprotocol.seichain.confidentialtransfers;
 
 import "gogoproto/gogo.proto";
 import "google/api/annotations.proto";
+import "confidentialtransfers/confidential.proto";
 
 option go_package = "github.com/sei-protocol/sei-chain/x/confidentialtransfers/types";
 
-// TODO: Define any query messages here
 // Query defines the gRPC querier service.
 service Query {
-  // TODO: This is a mock method. Remove when we add real queries
-  rpc TestQuery (TestQueryRequest) returns (TestQueryResponse) {
+  rpc GetAccount (GetAccountRequest) returns (GetAccountResponse) {
     option (google.api.http) = {
-      get: "/seichain/confidentialtransfers/test_query"
+      get: "/seichain/confidentialtransfers/account/{address}/{denom}"
     };
   }
+
+  rpc GetAllAccounts (GetAllAccountsRequest) returns (GetAllAccountsResponse) {
+    option (google.api.http) = {
+      get: "/seichain/confidentialtransfers/account/{address}"
+    };
+  }
+}
+
+message GetAccountRequest {
+  string address = 1;
+  string denom = 2;
+}
+
+message GetAccountResponse {
+  CtAccount account = 1;
 }
 
-// TODO: This is a mock method. Remove when we add real queries
-message TestQueryRequest {
-  string test_field = 1;
+message GetAllAccountsRequest {
+  string address = 1;
 }
 
-// TODO: This is a mock method. Remove when we add real queries
-message TestQueryResponse {
-  string test_field = 1;
+message GetAllAccountsResponse {
+  repeated CtAccount accounts = 1;
```

Comment:
CtAccount itself does not have reference to the denom associated with the account - we should make sure this query responds with the names of the denoms that the address has accounts for

(actually that's probably the most important thing for this query, I think it's mostly useful to let users know what denoms some account has an confidential transfer accounts for, the user can then query that account directly to get the account details)